consoles/sshVirtsh.pm

```diff
@@ -524,6 +524,10 @@ sub resume {
     bmwqemu::diag "VM " . $self->name . " resumed";
 }
 
+sub get_remote_vmm {
```

Comment:
This function looks odd. It is not present in os-autoinst (at least not the current master) but you also don't use it anywhere in the added lines of code.

Reply:
`get_remote_vmm()` is to be used from test code (see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6991). (Forgot to add this information to the description.)